src/openforms/authentication/base.py

```diff
@@ -81,6 +81,10 @@ def get_provides_auth(self) -> List[str]:
         else:
             return list(self.provides_auth)
 
+    @property
+    def is_enabled(self) -> bool:
+        return True
```

Comment:
Please make this a class attribute instead, just like `is_demo_plugin`

The method + `@property` route can then be used in actual plugins to make it dynamic.